images/os/Dockerfile

```diff
@@ -7,7 +7,10 @@ RUN yum install -y ostree yum-utils selinux-policy-targeted && \
     yumdownloader --destdir=/tmp/rpms origin-node origin-clients && \
     for i in $(find /tmp/rpms/ -name origin-* -iname *.rpm); do echo "Extracting $i ..."; rpm2cpio $i | cpio -div; done && \
     mv etc usr/ && \
-    ostree --repo=/srv/repo commit --parent=$commit --tree=ref=$commit --tree=dir=. --selinux-policy / \
+    mkdir -p /var/tmp/tmprootfs/etc && \
+    ostree checkout $commit --subpath /usr/etc/selinux /var/tmp/tmprootfs/etc/selinux && \
```

Comment:
use `/tmp` unless there's a reason not to

Reply:
Yeah, that's just a habit of working around `/tmp` possibly being on tmpfs, which has some peculiarities around xattrs. Although... yeah, it should be fine in this case. Will change!